graviti/utility/request.py

```diff
@@ -9,10 +9,10 @@
 from uuid import uuid4
 
 from requests.models import Response
-from tensorbay.utility import config, get_session
 
 from graviti.__version__ import __version__
 from graviti.exception import RESPONSE_ERROR_DISTRIBUTOR, ResponseError
+from graviti.utility import config, get_session
```

Comment:
graviti.utility.requests

Reply:
I have imported these two objects in utility.__init__.py, is it right?

Reply:
done